app/views/admin/depositors_search/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<% content_for :breadcrumbs do %>
```

Comment:
Can new and index be combined into a single form?

Reply:
That's what I'll try to do.